flashinfer/gdn_kernels/delta_rule_dsl/__init__.py

```
@@ -0,0 +1,12 @@
+try:
+    from .delta_rule_sm90 import (
+        delta_rule_prefill_dsl_sm90 as chunk_gated_delta_rule_sm90,
+    )
+except ImportError:
+    chunk_gated_delta_rule_sm90 = None  # type: ignore
+
+
+__all__ = [
+    "chunk_gated_delta_rule_sm90",
+    "_has_sm90_delta_rule_dsl",
+]
```

Ruff (0.15.15)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)